### DISCUSSION TITLE
I can't access the dashboard because I'm not logged in.

### DISCUSSION THREAD

**Discussion by ParkYoungWoong** [supabase/supabase#18573] [Questions]
I can't access the dashboard because I'm not logged in.(GitHub)
I can't log in for several days.
I thought things would get better, but I still can't log in.
I tried removing local storage and cookies, but still can't log in.
I don't know how to deal with this problem.

**Comment by GaryAustin1** [comment]
Hard to help with no error information.
You probably should contact support at support@supabase.io .
